.github/workflows/publish.yml

```diff
@@ -70,7 +70,7 @@ jobs:
             echo "value=${{ github.ref_name }}" >> $GITHUB_OUTPUT
           else
             SHORT_SHA=$(echo "${{ github.sha }}" | cut -c1-7)
-            echo "value=${{ github.ref_name }}-${SHORT_SHA}" >> $GITHUB_OUTPUT
+            echo "value=${SHORT_SHA}" >> $GITHUB_OUTPUT
```

Comment:
If we are making this change, can we do so everywhere -- 

1. .github/workflows/container-build-test.yml
2. .github/workflows/e2e-test.yml
3. iirc, there should a couple more places in this file as well
4. .github/workflows/release.yml